go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -119,6 +120,9 @@ type OperationModel struct {
 	// operation to ensure a duplicate txn will not occur. See:
 	// https://bugzilla.redhat.com/show_bug.cgi?id=2042001
 	Name string
```

Comment:
is it possible to just change Name to store the interface{} and drive the NameField as well as the real name from that? (rather than introducing another variable that holds almost the same thing)

Reply:
@trozet Thanks! good thought, updated the PR just with existing ` Name` field.